nakama/nakama.lua

```diff
@@ -5,6 +5,9 @@ local log = require "nakama.util.log"
 local async = require "nakama.util.async"
 local api_session = require "nakama.session"
 
+local uri = require "nakama.util.uri"
+local uri_encode = uri.encode
+
```

Comment:
We use `uri_encode` in some functions
Issue https://github.com/heroiclabs/nakama-defold/issues/14